patterns/hidden-404.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * Title: 404
+ * Slug: twentytwentyfive/hidden-404
+ * Inserter: no
+ */
+?>
+
+<!-- wp:columns {"style":{"spacing":{"padding":{"top":"50px","bottom":"50px","left":"50px","right":"50px"},"blockGap":{"left":"22px"}}}} -->
```

Comment:
These values need to be replaced once the spacing preset are added in theme.json.
(for now all we need to do is wait)

Reply:
@cliffralessio a first iteration on the spacing presets has been [merged here](https://github.com/WordPress/twentytwentyfive/pull/49). Can you please adjust the different spacing values that are hardcoded and replace them with the presets?